### PR TITLE
Implement documents.getRegistration().

### PR DESCRIPTION
**Pull request by dmitrizagidulin**

### CODE REVIEW

File: lib/documents.js [hunk 1]
```diff
@@ -248,6 +228,33 @@ export async function _generateInternalId() {
...
 
+/**
+ * Retrieves the encrypted registration for an internalId.
+ *
+ * @param {object} options - Options to use.
+ * @param {string} options.internalId - The internalId to use.
+ *
+ * @returns {Promise<object>} Resolves with the registration JWE.
+ */
+export async function getRegistration({internalId}) {
```
Comment (dlongley) [inline lib/documents.js:239]:
```suggestion
export async function getRegistration({internalId} = {}) {
```

**Review by dlongley — APPROVED** [review]
Approved with one minor tweak required.

File: test/mocha/10-documents.js [hunk 2]
```diff
@@ -15,6 +15,43 @@ const recipients = [
...
+});
+
+const key2 = new X25519KeyPair({
+  id: 'did:key:z6MkrefS4sDAGNBdo7CeXKh52sBfK94NGMANfHKfbYpvPz8S' +
+    '#z6LScKCBLkDApcTvYPbjQi6EDKgpYwWiM9Ppd6X1PbjXF2dg',
+  controller: 'did:key:z6MkrefS4sDAGNBdo7CeXKh52sBfK94NGMANfHKfbYpvPz8S',
+  type: 'X25519KeyAgreementKey2019',
+  publicKeyBase58: 'e21pSQJj9kBT1Dxt4aGtjULhnybeYDfk7oKu95zXerv',
+  privateKeyBase58: 'bcB3uZng7RPz7VSEJSid54cyiU2STGHk4Ub91VEenPP'
+});
```
Comment (mattcollier) [inline test/mocha/10-documents.js:34]: Please put this into mock.data.  Pattern to follow is demonstrated here: https://github.com/digitalbazaar/bedrock-account/tree/master/test/mocha

File: test/mocha/10-documents.js [hunk 3]
```diff
@@ -15,6 +15,43 @@ const recipients = [
...
+
+describe('documents.getRegistration()', () => {
```
Comment (mattcollier) [inline test/mocha/10-documents.js:36]: Would it be easy enough to add a test for the NotFoundError?

**Pull request merged**